### PR TITLE
Add board

### PR DESCRIPTION
**Pull request by lrharris215**
Making a GET request to /boards will return both the Player Board and the Computer Board. The boards are stored in a Board Repository that only persists while the server is running.

### CODE REVIEW

File: src/main/java/com/battleship/backend/BoardRepository.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+public class BoardRepository {
+    Board[] boards;
+    Board playerBoard;
+    Board computerBoard;
+
+    public BoardRepository(){
+        playerBoard = new Board("Player Board");
+        computerBoard = new Board("Computer Board");
+        boards = new Board[] { playerBoard, computerBoard};
+    }
```
Comment (kudukacar) [inline src/main/java/com/battleship/backend/BoardRepository.java:16]: Do you have a Java linter?  I see some spacing issues here.

**Review by kudukacar — COMMENTED** [review]
Looks great!  Saw some spacing issues - recommend installing a Java linter

File: src/test/java/com/battleship/backend/BattleshipApplicationTests.java [hunk 2]
```diff
@@ -15,19 +16,21 @@
...
+	void testWelcomeControllerExists() throws Exception {
+		assertThat(welcomeController).isNotNull();
 	}
 
 	@Test
-	void testWelcomeControllerExists() throws Exception {
-		assertThat(welcomeController).isNotNull();
+	void testBoardControllerExists() throws Exception {
+		assertThat(boardController).isNotNull();
 	}
```
Comment (kudukacar) [inline src/test/java/com/battleship/backend/BattleshipApplicationTests.java:34]: I'm not following the purpose of these tests.  Why do we have these tests?  If you have the relevant controller tests, why do you need these tests?

File: src/main/java/com/battleship/backend/BoardRepository.java (hunk 1 above)
Comment (lrharris215) [reply src/main/java/com/battleship/backend/BoardRepository.java:16]: I've just been using the built in IntelliJ linter. Looking at CheckStyle now.

Comment (kudukacar) [reply src/main/java/com/battleship/backend/BoardRepository.java:16]: Ok.  I'm surprised intellij allowed the above spacing.  Don't spend too much time on a linter.  Move on if you can't get one to work.

**Review by kudukacar — APPROVED** [review]
Looks good!

**Pull request merged**